packages/rosetta/configuration/send_funds.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -e
+
+#
+# Usage:
+# ./send_fund.sh <destination>
```

Comment:
Note that `ADDR` must be set in the environment.  Did you mean to use `DESTINATION` where you use `ADDR` below? The invocation in `faucet.py` does not seem to set `ADDR`.

Reply:
whoops.. they were only used in output messages and in the filenames... which happened to be luckily unimportant... fixing